src/main/java/com/mparticle/kits/AppsFlyerKit.java

```diff
@@ -38,6 +38,7 @@
 public class AppsFlyerKit extends KitIntegration implements KitIntegration.EventListener, KitIntegration.AttributeListener, KitIntegration.CommerceListener, AppsFlyerConversionListener, KitIntegration.ActivityListener {
 
     private static final String DEV_KEY = "devKey";
+    private static final String APPSFLYERID_INTEGRATION_KEY="appsflyer_id_integration_setting";
```

Comment:
can you space this out:
`APPSFLYERID_INTEGRATION_KEY = "appsflyer_id_integration_setting"`